### DISCUSSION TITLE
Selecting Specific Directories to Synch

### DISCUSSION THREAD

**Discussion by GeekBone** [abraunegg/onedrive#1967] [General]
No matter what I put in the config file or what command I use, it just seems like whenever I synchronize, it syncs the entire onedrive. I used the following:
```
onedrive --synchronize --single-directory 'Current Projects' / 'Education' /  'Game Development' / 'Renpy' / 'Projects' --download-only 
```

and it downloaded everything in my OneDrive.

What am I doing wrong?

**Comment by abraunegg** [comment]
@GeekBone 
Lets start with some basics:
* How did you install the application?
* What is your application version: `onedrive --version`

Secondly, you are not using the options correctly. Please re-read the documentation.

If your command above is exactly as you have used it .. then, it would only be trying to sync 'Current Projects' and that *entire* tree.

You should be specifying the command as follows:
**Test your command:**
```
onedrive --synchronize --verbose --single-directory 'Current Projects/Education/Game Development/Renpy/Projects' --download-only --dry-run
```

**Run your command:**
```
onedrive --synchronize --verbose --single-directory 'Current Projects/Education/Game Development/Renpy/Projects' --download-only
```
